cardshifter-core/src/main/java/com/cardshifter/core/groovy/EffectDelegate.groovy

```diff
@@ -257,10 +270,9 @@ class EffectDelegate {
                     })
                 }
             }
-            assert closure : "$description: Unknown target $who"
+            assert closure : "Unknown target $who"
```

Comment:
Is removing the description from this error string a good idea? I imagine that when you have several files and lots of lines, you want contextual information to help you figure out where the error is, can/should we provide more detail here?